src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_1 = "supply";
+    private static final String OPERATION_2 = "buy";
+    private static final String OPERATION_AND_AMOUNT_SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        File toFile = new File(toFileName);
```

Comment:
It's unnecessary to create `File` when you can use `BufferedReader` only with `fileName`